src/components/static-map.js

```diff
@@ -307,12 +307,7 @@ export default class StaticMap extends PureComponent<StaticMapProps, State> {
         <div key="map-mapbox" ref={this._mapboxMapRef} style={mapStyle} className={className} />
         {/* AutoSizer is a pure component and does not rerender when map props change */}
         {/* rebind the callback so that it's triggered every render pass */}
-        <AutoSizer
-          key="autosizer"
-          disableWidth={Number.isFinite(width)}
-          disableHeight={Number.isFinite(height)}
-          onResize={this.props.onResize}
-        >
+        <AutoSizer key="autosizer" onResize={this.props.onResize}>
           {this._renderOverlays.bind(this)}
```

Comment:
Unrelated, not sure if pre-binding methods like this is still worthwhile?

Reply:
This is needed because `AutoSizer` only rerenders its children on size change, but  overlays need to be rerendered when any other prop changes. Rebinding on every render forces `AutoSizer` to call the function again.